<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameOver</class>
 <widget class="QDialog" name="GameOver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>704</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>704</width>
    <height>704</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>704</width>
    <height>704</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="GameOverLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>704</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="RoundsLabel">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>352</y>
     <width>96</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 241, 232);
font: 32pt '8-bit Arcade In';</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QLabel" name="NormalLabel">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>408</y>
     <width>96</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 241, 232);
font: 32pt '8-bit Arcade In';</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QLabel" name="BossLabel">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>472</y>
     <width>96</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 241, 232);
font: 32pt '8-bit Arcade In';</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QLabel" name="FoodLabel">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>536</y>
     <width>96</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 241, 232);
font: 32pt '8-bit Arcade In';</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
